### DISCUSSION TITLE
Turret Computer change

### DISCUSSION THREAD

**Discussion by thatisjake** [tsunamayo/Starship-EVO#4657] [Suggestions]
I think that when you hook up switch or any logic to a turret computer, it should deactivate the turret. (Switch off, turret off) This would allow you to return the turret to resting position by the flip of a switch. 
Retractable turrets would be much cooler, as things like this could be possible https://www.youtube.com/watch?v=cM49l9-oFTk

**Comment by Gallt** [comment]
100% agree, was gonna suggest the same thing. Would love to be able to make sure my "safety is on" from a simple and single button without worrying about weapon group switching.
